code/src/index.css

```diff
@@ -1,13 +1,183 @@
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+    background-color: #191414;  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* styling for main section */
+.main-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+/* styling for header */
+.header-container {
+  margin: 0px 20px;
+  grid-column: span 4;
```

Comment:
Glad to see you guys using ` grid-column: span 4 ;`, a way of building layout that I seldom use.